build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -9,9 +9,9 @@ import { readJsonSync } from "fs-extra";
 import { Project, SourceFile } from "ts-morph";
 import { BrokenCompatTypes } from "../common/fluidRepo";
 import { PackageJson } from "../common/npmPackage";
+import { typeOnly } from "../typeValidator/compatibility";
```

Comment:
Move this file because its contents are mostly related to the core validator logic, not the CLI.